### ISSUE TITLE
Entities returning no results

### ISSUE THREAD

**Issue by DoughnutAus** [MscrmTools/MscrmTools.AttributeUsageInspector#8]
Hi, 
We've been using the Attribute Usage Inspector and it returns results for some Entities but not others. 

It appears to be working and brings up the screen where it says "Loading usage data..." but then it stops and we see no results. On some other Entities it appears to be working fine. 

Is there a reason it's not working on some Entities and are there any settings we can change to get it working for all Entities?

Thanks

**Comment by MscrmTools** [comment]
It shouldn’t...
Is the problematic entity is a custom one or a standard for one?

**Comment by DoughnutAus** [comment]
The Entities it's not returning results for are Contacts, Leads and Opportunities - all standard Entities that come Out of the Box

**Comment by MscrmTools** [comment]
There is some settings you can try to play with by putting lower values on number of records and attributes to retrieve for one request

**Comment by DoughnutAus** [comment]
What do the number of records per call and number of attributes per call refer to. I lowered them but it made no difference. I also filtered to only certain fields and still didn't work. Do we have too many records for it to search on?

**Comment by MscrmTools** [comment]
Possibly.
Lowering the settings might produce lighter requests. 
I will check the code tomorrow to see if there is a lack in error handling that could explain it stops without notice

**Comment by MscrmTools** [comment]
Ok, so there was a lack of error handling...
Can you test this version and tell me what is the error message?
[MscrmTools.AttributeUsageInspector.zip](https://github.com/MscrmTools/MscrmTools.AttributeUsageInspector/files/2118250/MscrmTools.AttributeUsageInspector.zip)

**Comment by DoughnutAus** [comment]
I copied the .dll into the Plugins folder and tried again, but I still
didn't get an error message. For example, I clicked on Contacts, it
showed the Loading data usage... screen (see Loading data usage
image) and then after a few seconds it stopped and I get the blank
screen (see Contacts_Blank image). I tried lowering the records per
call and attributes per call and filtered for less attributes and
standard attributes etc., but I still get a blank screen. It works
fine on Accounts and some other Entities.

----- Original Message -----
